README.md

```diff
@@ -32,6 +32,8 @@ Reviewing code is hard, and as the author of code you should try your best to ma
 
 Read the page on [Writing PRs](https://github.com/dimagi/code-review/blob/es/writing-prs/Writing_PRs.md)
 
+Also look at Google's guide to [getting through code reviews](https://google.github.io/eng-practices/review/developer/)
```

Comment:
I think it's possible we could even say something a bit stronger to encourage new people to look at this. E.g. "we mostly follow Google's guide with minor practical changes around some expectations". But maybe that's a bit too strong for now.

Reply:
Hmm, I agree with that, and it didn't seem controversial in today's discussion. I'll merge this PR and open another with some stronger recommendations.